yamllintrc

```diff
@@ -0,0 +1,29 @@
+extends: default
+
+ignore:
+  - '**/hotsos/tests/unit/fake_data_root/**'
+  - '**/.tox/**'
+  - '**/examples/**'
+  - '**/.github/**'
+
+rules:
+  # https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.line_length
```

Comment:
should be https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.document_start